formatters/api.go

```diff
@@ -21,7 +21,7 @@ var (
 	}))
 	// Default HTML formatter outputs self-contained HTML.
 	htmlFull = Register("html", html.New(html.Standalone(), html.WithClasses())) // nolint
-	Svg      = Register("svg", svg.New())
+	Svg      = Register("svg", svg.New(svg.EmbedFont("Liberation Mono", svg.FontLiberationMono, svg.WOFF)))
```

Comment:
This should be `SVG =` to comply with `golint`. I'm a bit surprised it didn't complain TBH.

Reply:
renamed